x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/SemanticSearchAction.java

```diff
@@ -111,7 +112,7 @@ public static Request parseRestRequest(RestRequest restRequest) throws IOExcepti
             return builder.build();
         }
 
-        private final String[] indices;
+        private String[] indices;
```

Comment:
IIUC, `indices` could potentially be `null` which is undesirable. I suggest we default it to an empty string array both here and in the Builder.

Reply:
Sorry I forgot to push the `Objects.requireNonNull` in the ctor and the builder ctor